package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "docker": "docker-compose run --rm ccxt",
     "build": "npm run build-without-docs && npm run pandoc-all && npm run update-badges",
-    "build-without-docs": "npm run export-exchanges && npm run vss && npm run check-js-syntax && npm run browserify && npm run transpile && npm run qa",
+    "build-without-docs": "tsc && npm run export-exchanges && npm run vss && npm run check-js-syntax && npm run browserify && npm run transpile && npm run qa",
```

Comment:
Oops, I've added it to .travis.yml... should probably delete it from there and add typescript to the deps in `package.json` then...